src/views/pages/ranking.html

```diff
@@ -69,12 +69,12 @@ <h1 ng-if="stage.rounds > 0">
                     </tr>
                 </thead>
                 <tbody>
-                    <tr ng-repeat="item in scoreboard[stage.id] | index | orderBy:stage.sort:stage.rev track by item.team.number">
+                    <tr ng-repeat="item in scoreboard[stage.id]">
```

Comment:
sorting is now broken?

Reply:
Yes. It's brought back in Itay Dafner's PR

Reply:
Apparently it was broken in the 'small refactoring' PR, and I didn't see it due to caching issues.
I've pushed a fix on master. Just cherry-pick that one for now, and leave unrelated stuff like this out of the PR.

Reply:
Sure thig